cloudfoundry/resource_cf_user.go

```diff
@@ -314,7 +318,35 @@ func resourceUserUpdate(ctx context.Context, d *schema.ResourceData, meta interf
 func resourceUserDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	session := meta.(*managers.Session)
 	id := d.Id()
-	err := session.ClientUAA.DeleteUser(id)
+
+	endpoint := fmt.Sprintf("/v2/users/%s", id)
+	req, err := session.RawClient.NewRequest("DELETE", endpoint, nil)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	resp, err := session.RawClient.Do(req)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	defer func() {
+		err := resp.Body.Close()
+		if err != nil {
+			panic(err)
```

Comment:
`panic()` might cause stale state locks so best to just error out

Reply:
you are right, I fix change that right away

Reply:
resolved in 3a17a25